src/test/java/teammates/test/cases/ui/browsertests/AdminActivityLogPageUiTest.java

```diff
@@ -57,6 +57,12 @@ public void testViewActionsLink(){
              * This can happen if this test is run right after the server is started.
              * In this case, no view actions can be done.
              */
+        } catch (IndexOutOfBoundsException invisibletmtlogs) {
+            /*
+             * This can happen if all the log entries are from test accounts
+             * (i.e emails ending with .tmt) because they are invisible.
+             * In this case, no view actions can be done.
+             */
```

Comment:
Can go ahead and merge. But before that, may be rename the two exceptions to the more conventional names nsee and ioobe? The current names, while more informative, do not match the object type, which can look odd if we decide to do more things with the exception object. For example, `emptylogs.getMessage()` looks misleading because the it is odd to have an exception called `emptylogs`. `excpetionFromEmptyLogs` is more accurate, but a bit long (I don't mind it though). You choose :-)